# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1304721633 14400
#      Fri May 06 18:40:33 2011 -0400
# Branch stable
# Node ID 5fd90b7cb78761f85afdb8b91db1f0b1fc663f8a
# Parent  c6378cec77af0b406ccf9a3c114f64022957ab18
doc: Update contributors list.

* contributors.in: Add Daniel Kraft to the list.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -106,16 +106,17 @@ Joel Keay
 Mumit Khan
 Paul Kienzle
 Aaron A. King
 Arno J. Klaassen
 Geoffrey Knauth
 Heine Kolltveit
 Ken Kouno
 Kacper Kowalik
+Daniel Kraft
 Oyvind Kristiansen
 Piotr Krzyzanowski
 Volker Kuhlmann
 Tetsuro Kurita
 Miroslaw Kwasniak
 Rafael Laboissiere
 Kai Labusch
 Claude Lacoursiere
